src/test/java/org/lightcouch/tests/ViewsTest.java

```diff
@@ -95,6 +96,18 @@ public void byComplexKey() {
 	}
 
 	@Test
+	public void byComplexKeys() {
+		List<int[]> keysToGet = new Vector<int[]>();
```

Comment:
I know this is just a test and doesn't need changing. But in Java you're generally better off using an `ArrayList` rather than a Vector because most code doesn't need the synchronization that Vector provides.